backend/models/users.py

```diff
@@ -12,18 +12,29 @@ class Role(db.Model):
 
 
 class User(db.Model):
-
-    """This class is used for user table in database."""
+    """User table in the database"""
 
     __tablename__ = 'users'
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.Text)
     alias = db.Column(db.Text)
     email = db.Column(db.Text)
-    password = db.Column(db.Text)
+    _password = db.Column(db.Text, nullable=False)
```

Comment:
I'd rather do not change field name. It's not common to use underscore prefixes for DB column names.

Reply:
the idea was to notify coders that passwords will be stored with hashing (not in a raw state). I've seen such things in examples, so coders are aware that some additional stuff will happen with passwords. But if it's required i can change the name

Reply:
I'm ok to have object's attribute `_password`, but field with the same name will be created in DB, what is not good.

Reply:
consensus